demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -397,7 +409,7 @@ def is_file_valid_in_set(self, file_path, file_type, ignored_errors=None):
                 is_valid = self._is_mapper_incident_types_found(mapper_data)
             elif file_type == constants.FileType.PLAYBOOK:
                 playbook_data = get_playbook_data(file_path)
-                is_valid = self._are_playbook_entities_versions_valid(playbook_data, file_path)
+                is_valid, error = self._are_playbook_entities_versions_valid(playbook_data, file_path)
```

Comment:
why do you need to return the error?
it is not used
```suggestion
                is_valid, _ = self._are_playbook_entities_versions_valid(playbook_data, file_path)
```